MODULE.bazel

```diff
@@ -11,7 +11,7 @@ register_toolchains(
 )
 
 bazel_dep(name = "platforms", version = "0.0.4")
-bazel_dep(name = "rules_license", version = "0.0.7")
+bazel_dep(name = "rules_license", version = "1.0.0")
```

Comment:
Dow e want to _force_ 1.0.0 on people? I'm only asking because `MODULE.bazel` should declare the minimum version needed.

Reply:
It should be fine to update everyone to use 1.0.0: it's not hugely different from the previous release and I don't think it introduces new dependencies. OTOH, it includes the `purl` attribute in the `PackageInfo` which is useful when generating SBOMs